data/amd/register/msr/MSRC001_0055.yml

```diff
@@ -0,0 +1,22 @@
+- name: MSRC001_0055
+  long_name: "Reserved"
+  purpose: |
+       "
+        MSRC001_0055 [Reserved.] (Core::X86::Msr::IntPend)
+       "
+  size: 64
+  arch: amd64
+
+  access_mechanisms:
+      - name: rdmsr
+        is_read: True
+        address: 0xC0010055
+
+      - name: wrmsr
```

Comment:
The description of this register says "Read Only", so delete the wrmsr access mechanism